internal/commands/app/create_test.go

```diff
@@ -269,7 +272,7 @@ func TestAppCreateHandler(t *testing.T) {
 				DataSource: "test-cluster"},
 		}
 
-		assert.Nil(t, cmd.Handler(profile, ui, cli.Clients{Realm: client}))
+		assert.Nil(t, cmd.Handler(profile, ui, cli.Clients{rc, ac}))
```

Comment:
I'm adding a 3rd client with hosting changes, do you mind using the Field names here?